manage.py

```diff
@@ -241,6 +241,21 @@ def toggle_site_status():
         print('Done!')
 
 
+@cli.command(name='set_rate_limits')
+@click.argument('per_ip')
+@click.argument('window_size')
+def set_rate_limits(per_ip, window_size):
+    """ Set rate limit parameters for the AcousticBrainz webserver
+
+    Args:
+        per_ip (int): the number of requests allowed per IP address
+        window_size (int): the window in number of seconds for how long the limit is applied
+    """
+    if per_ip / float(window_size) <= 1:
```

Comment:
this doesn't have the full functionality of the original script - it doesn't print current limits if no arguments are given